### ISSUE TITLE
apt install python twice

### ISSUE THREAD

**Issue by jack-gits** [pytorch/serve#1982]
### 🐛 Describe the bug

I found there're two commands to install the python3.8 in docker/Dockerfile.
is it duplicated?
screenshots as below

![image](https://user-images.githubusercontent.com/30545972/202886701-c242b380-a5a7-4ed9-820e-67f6c9983c75.png)

![image](https://user-images.githubusercontent.com/30545972/202886756-ce5a6426-f980-4fcb-84ab-9cc22cc63161.png)

### Error logs

None

### Installation instructions

None

### Model Packaing

None

### Versions

None

### Repro instructions

None

**Comment by msaroufim** [comment]
You're right, thanks for spotting this - would you like to make the PR to fix it? Just build the image once to sanity check that it worked and attach the logs to PR. Should be easy to merge

**Comment by jack-gits** [comment]
Let me try.
